src/ast/nodes/MemberExpression.ts

```diff
@@ -93,32 +95,43 @@ export default class MemberExpression extends NodeBase {
 			}
 		} else {
 			super.bind();
+			if (this.propertyKey === null) this.updatePropertyKey();
```

Comment:
`analysePropertyKey` for consistency again? Whatever we choose, the main thing is really having a consistent verb.